docs/strings.rst

```diff
@@ -177,5 +177,5 @@ it will be used as-is; Unicode strings will be encoded down to UTF-8.
 In the file, h5py uses the most-compatible representation; H5T_CSET_ASCII for
 characters in the ASCII range; H5T_CSET_UTF8 otherwise.
 
-    >>> grp = f.create_dataset(b"name")
```

Comment:
The `b` should stay - it's illustrating that you can pass byte strings in for names in the file.